lib/nimble_options.ex

```diff
@@ -559,7 +567,12 @@ defmodule NimbleOptions do
 
   defp validate_option({opts, orig_opts}, key, schema) do
     with {:ok, value} <- validate_value({opts, orig_opts}, key, schema),
-         {:ok, value} <- validate_type(schema[:type], key, value) do
+         # NOTE: This is just for testing purposes. Will be removed.
+         {:ok, value} <-
+           if(schema[:type] == :integer,
+             do: validate_type(schema[:type], key, value, schema[:redact]),
+             else: validate_type(schema[:type], key, value)
```

Comment:
This is here only for the initial proof-of-concept. I've only implemented support for redacting `:integer` types for now.